service/src/intTest/java/uk/nhs/adaptors/gp2gp/common/storageconnectors/StorageConnectorTest.java

```diff
@@ -0,0 +1,49 @@
+package uk.nhs.adaptors.gp2gp.common.storageconnectors;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+public class StorageConnectorTest {
+
+    @Autowired
+    private StorageConnector storageConnector;
+
+    private static final String FILE_EXPECTED_DOWNLOAD = "src/intTest/resources/downloads/test.txt";
```

Comment:
Should use a classpath reference rather than a relative file path

Reply:
This is an expected url for a downloaded file which is not yet present. Therefore classpath reference will cause a null pointer when attempting to construct the URI. We could treat is as an assert value.